node/cmd/guardiand/node.go

```diff
@@ -244,130 +244,129 @@ func init() {
 	guardianKeyPath = NodeCmd.Flags().String("guardianKey", "", "Path to guardian key (required)")
 	solanaContract = NodeCmd.Flags().String("solanaContract", "", "Address of the Solana program (required)")
 
-	ethRPC = NodeCmd.Flags().String("ethRPC", "", "Ethereum RPC URL")
+	ethRPC = NodeCmd.Flags().String("ethRPC", "", "Ethereum RPC URL.\nFormat: 'ws://'. Example: 'ws://eth-devnet:8545'")
```

Comment:
Does it matter if the format is 'ws://' or 'wss://'?  If we only show 'ws://' are we implying 'wss://' is not allowed?  For one testnet guardian 'wss://' was used here.
This comment applies everywhere.